components/prize_pool/commons/prize_pool_legacy.lua

```diff
@@ -28,6 +29,8 @@ local CACHED_DATA = {
 	qualifiers = {},
 }
 
+local CHECKMARK = '<div class="fa fa-check green-check" style="cursor: auto;"></div>'
```

Comment:
```suggestion
local CHECKMARK = '<div class="fa fa-check green-check"></div>'
```
See https://github.com/Liquipedia/Lua-Modules/pull/1664#issuecomment-1211742761

Reply:
Yeah dota2 did that, aoe dosen't. I think it makes more sense not to have it, so have removed it.